src/main/resources/application.properties

```diff
@@ -9,3 +9,4 @@ spring.rabbitmq.listener.retry.max-interval=35000
 biosamples.client.aap.username=${biosamplesClientAAPUsername}
 biosamples.client.aap.password=${biosamplesClientAAPPassword}
 aap.url=https://explore.api.aap.tsi.ebi.ac.uk/auth
+logging.level.uk.ac.ebi.biosamples.client.utils=WARN
```

Comment:
Do we want it to commit?

Reply:
Yes, very much so. This is where the 16Gb log file originated from.